app/src/main/res/layout/activity_main.xml

```diff
@@ -2,24 +2,32 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/white"
+    android:id="@+id/coordinateLayout"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <FrameLayout
-        android:id="@+id/frameContainer"
+    <LinearLayout
         android:layout_width="match_parent"
-        android:background="@color/white"
-        android:layout_marginBottom="?attr/actionBarSize"
         android:layout_height="match_parent"
-        app:layout_behavior="@string/appbar_scrolling_view_behavior" />
+        android:orientation="vertical">
 
-    <android.support.design.widget.BottomNavigationView
-        android:id="@+id/navigation"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_gravity="bottom"
-        android:background="?android:attr/windowBackground"
-        app:itemBackground="@color/white"
-        android:foreground="?attr/selectableItemBackground"
-        app:itemTextColor="@color/colorPrimaryDark"
-        app:menu="@menu/navigation" />
+        <FrameLayout
+            android:id="@+id/frameContainer"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:background="@color/white"
```

Comment:
android:color

Remove color/white

Reply:
android:color used to change the text color actually. android:background used for background changes

Reply:
android:color/white